odkx-src/sync-endpoint-cloud-setup.rst

```diff
@@ -360,10 +364,26 @@ Connecting to your virtual machine
 
   .. code-block:: console
 
-    $ docker stack ls
+    $ sudo docker stack ls
 
-  If there are 8 (or 7 without https) services running under the name
-  `syncldap`, everything is running properly.
+  To see all of the Docker processes/containers that are actively running, use the following command:
+
+  .. code-block:: console
+
+    $ sudo docker ps
+
+  .. image:: /img/setup-azure/azure-connecting-to-virtual-machine2.png
+    :width: 600
+
+  If there are 9 (or 8 without https) services running under the name
+  `syncldap`, everything is running properly. The following command shows all (both stopped and running) containers/processes:
+
+  .. code-block:: console
+
+    $ sudo docker ps -a
+
+  .. image:: /img/setup-azure/azure-connecting-to-virtual-machine3.png
```

Comment:
Is this whole screen shot the processes? Or is there something specific to box/highlight/point to?

Reply:
Yes, this screen shot shows 3 different commands and the probable inputs. I will highlight the commands to make it more clear.